src/surfaces/actionBar/actionBarActionsButtonDrawerOption.jsx

```diff
@@ -59,7 +59,8 @@ class ActionBarActionsButtonDrawerOption extends React.PureComponent {
                 }
             }
 
-            option.onClick(onDrawerToggle);
+            onDrawerToggle();
+            option.onClick();
             return false;
```

Comment:
Are the Boolean return values actually meaningful?
If not, I would like to see them all changed to just `return;` where we need to bail out early, and then remove the final one on Line 68.

Thoughts?

_**Not a Blocker**_

Reply:
Yeah, let's please do that, if you would be so kind.

Reply:
Sure

Reply:
The change looks good, but are we waiting for return change before merging this?

Reply:
Ready :)

Reply:
Not sure if I understand the return change @cbrewer1689  I can refactor the function to avoid using return. is that what you mean?

Reply:
@cletux87 I think @cbrewer1689 was referring to exactly the work you did in [e248d81](https://github.com/saddlebackdev/react-cm-ui/pull/359/commits/e248d81b3a627cf2528f88147ddfd9fb98c2a0e0).  So, it's done!  I think we can merge this if @cbrewer1689 agrees.